packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -83,7 +83,7 @@ function FeedNav(): ReactElement {
         ))}
       </TabContainer>
       {isMobile && sortingEnabled && isSortableFeed && (
-        <div className="sticky flex h-11 w-20 -translate-y-12 translate-x-[calc(100vw-100%)] items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
+        <div className="fixed flex h-11 w-20 -translate-y-12 translate-x-[calc(100vw-100%)] items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
```

Comment:
I don't think we even need fixed here. Maybe `absolute` is enough?

Reply:
`fixed` and `absolute` won't work.

Reply:
Can you elaborate on the primary difference, UI-wise?

Reply:
Yeah, I've tried numerous times as well. Though from your screenshots, gradient seems to be present on both.